governance/polls/Ratification Poll for Modify Core Unit Budget, MKT-001 (MIP40c3-SP33) - November 8, 2021.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Ratification Poll for Modify Core Unit Budget, MKT-001 (MIP40c3-SP33) - November 8, 2021
+summary: MIP40c3-SP33 adds the budget for Core Unit MKT-001: Content Production for December 2021 through May 2022.
+discussion_link: https://forum.makerdao.com/t/mip40c3-sp33-modify-core-unit-budget-mkt-001/10150
+vote_type: Plurality Voting
+categories:
+   - MIPs
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2021-11-08T16:00:00
+end_date: 2021-11-22T16:00:00
+---
+# Ratification Poll for Modify Core Unit Budget, MKT-001 (MIP40c3-SP33) - November 8, 2021
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 8 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP40c3-SP33 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP40c3-SP33 adds the budget for Core Unit MKT-001: Content Production for December 2021 through May 2022. We are requesting a 6-month budget allocation of 424,944 DAI.*
+
+Please review the links below to inform your position on this proposal before voting.
+* [Cannonical Proposal Version](https://github.com/makerdao/mips/blob/1e68595057de5e9d2154a3320ff121ea7ad1f94a/MIP40/MIP40c3-Subproposals/MIP40c3-SP33.md)
```

Comment:
Just made a correction. Use this commit instead: https://github.com/makerdao/mips/blob/06a913382e723d202afb586be6fb09f3a9915085/MIP40/MIP40c3-Subproposals/MIP40c3-SP33.md